src/Form.scss

```diff
@@ -25,19 +30,52 @@ fieldset {
   }
 }
 .checkbox-container {
-  label { margin-left: 0.5em;}
+  position: relative;
+  display: flex;
+  align-items: center;
+
+  label {
+    margin: 0 0.25em 0 0.5em;
+  }
 }
 
 input[type=checkbox] {
   margin: 0;
 }
 
 .input-group {
+  position: relative;
+
   label {
     display: inline-block;
   }
 }
 
+.integer-input {
+  .fa-circle-info {
+    margin-left: 0.25em;
+  }
+}
+
+.fa-circle-info {
+  color: #E0F1FF; // a slightly more saturated aliceblue, complementing the footer
+  outline: 1px solid #666666;
```

Comment:
This outline/border radius thing doesn't look as clean as the actual font circle unfortunately. I think it's better to just do a color and no background/outline/border radius. I came up with `#98b3c0` playing with color pickers

Reply:
`#98b3c0` with white 'foreground' (the `i` of the icon) fails accessibility due to low contrast: https://webaim.org/resources/contrastchecker/?fcolor=FFFFFF&bcolor=98B3C0

Reply:
Here's a grayscale version that has passing accessible contrast:
<img width="261" alt="Screenshot 2024-03-12 at 11 09 17 AM" src="https://github.com/alenia/planned-pooling/assets/284712/9dc2432d-6c8d-4e8b-82b3-cc13995ab51e">
https://webaim.org/resources/contrastchecker/?fcolor=333333&bcolor=D6D6D6

Reply:
I could also drop FontAwesome (limited free options, lighter-weight install) and get Material UI icons, which are more dependencies but all versions of the icons are free, and has a couple of nice variants; I like the [outlined variant](https://mui.com/material-ui/material-icons/?query=info&theme=Outlined&selected=InfoOutlined)